### DISCUSSION TITLE
[Feature] show people and photos of people in partner

### DISCUSSION THREAD

**Discussion by huanyang712** [immich-app/immich#11971] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Show people and photo of people in Partner Photos. Include an option in the Partner Sharing Settings to turn on or off.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5089
